task15.js

```diff
@@ -0,0 +1,9 @@
+// Write a function that takes a string and converts it to a number. If the conversion is not possible, return null.
+
+function convertToNumber(str) {
+  const convertedStr = isNaN(Number(str)) ? null : str;
```

Comment:
one small issue. When isNaN false, then u should return converted value. However now u return that value that u paste in function

```
const convertedValue = Number(str)
return isNaN(convertedValue ) ?  null  :  convertedValue  
```